### DISCUSSION TITLE
Error after upgrading to 4.20 - LOG: Encountered "\u001c" at line 1, column 70...

### DISCUSSION THREAD

**Discussion by pmcnallyma** [nextgenhealthcare/connect#5581] [General]
Hello,
I ran the update this morning on two Mirth environments and one is throwing an error saying this:
[2022-12-12 09:01:54.315]  ERROR  (org.apache.velocity.parser:196): LOG: Encountered "\u001c" at line 1, column 70.
Was expecting one of:
    "[" ...
    "|" ...
    "}" ...
    "}" ...
    

I can't figure out what to fix.  Googling says this may be related to APIs but we aren't using Mirth for those at this time.  
I tried viewing each channel to match up the times of the errors but I am unable to locate where this may be coming from.
Any advise or help would be appreciated.
Thanks,
P

**Comment by pacmano1** [comment]
That is a unicode character being seen in a string.   Likely been there all the time.  I suspect you need to find it and remove it.

**Reply by pmcnallyma** [reply]
Thanks.  I think this means the issue is in a transformer/filter and not the messages.  I wrote all of these channels but am not sure what JSON is.

**Reply by pacmano1** [reply]
Oh - might not be JSON actually, could just be the unicode character.

**Comment by pmcnallyma** [comment]
To close the loop, this was user (me) error.  I must have deleted the last } in the Subject line on our alert message.

**Comment by wi4seo** [comment]
Hi Everyone,

FREE Registration for Mirth Unconference 2023 is Now Opened!

To know more, please visit the event page and register yourself: https://wi4.org/mirth-connect-unconference-feb-2023/

Stay Tuned for More Updates!​

​Thanks
